java/src/jmri/jmrit/roster/JmritRosterBundle.properties

```diff
@@ -145,6 +145,7 @@ MediaRosterURLLabel = Web reference:
 MediaRosterAttributeName = Name
 MediaRosterAttributeValue = Value
 MediaRosterAttributeTableDescription = Roster custom attributes (delete Name to remove entry) (DO NOT CHANGE unless you know what you're doing)
+MediaRosterOpenSystemFileBrowserOnJMRIfnButtonsRessources=Open JMRI function button image ressource folder in file explorer
```

Comment:
I think it's good to have at least one space before and after the = sign.
`...Ressources = Open JMRI ...`

I think `Ressources` is spelled `Resources`.